app/inbound/process-submit-payment-request.js

```diff
@@ -0,0 +1,28 @@
+const db = require('../data')
+
+const getPaymentRequestByInvoiceNumber = require('./get-payment-request-by-invoice-number')
+const saveInvoiceNumber = require('./save-invoice-number')
+const savePaymentRequest = require('./save-payment-request')
+const saveInvoiceLines = require('./save-invoice-lines')
+
+const processSubmitPaymentRequest = async (paymentRequest) => {
+  const transaction = await db.sequelize.transaction()
+  try {
+    const existingPaymentRequest = await getPaymentRequestByInvoiceNumber(paymentRequest.invoiceNumber, transaction)
+    if (existingPaymentRequest) {
+      console.info(`Duplicate payment request received, skipping ${existingPaymentRequest.invoiceNumber}`)
+      await transaction.rollback()
+    } else {
+      await saveInvoiceNumber(paymentRequest.invoiceNumber, transaction)
+      await savePaymentRequest(paymentRequest, transaction)
+      const savedPaymentRequest = await getPaymentRequestByInvoiceNumber(paymentRequest.invoiceNumber, transaction)

```

Comment:
When you create a record using Sequelize you get a response of that saved object.  So rather than making a second call to the database here, you could update the `savePaymentRequest` function above to return that response.  Then you'll have access to the `paymentRequestId` you need.